lms/views/api/canvas/authorize.py

```diff
@@ -28,6 +28,7 @@ def authorize(self):
                         "response_type": "code",
                         "redirect_uri": self.request.route_url("canvas_oauth_callback"),
                         "state": CanvasOAuthCallbackSchema(self.request).state_param(),
+                        "scope": "url:GET|/api/v1/courses/:course_id/files url:GET|/api/v1/files/:id/public_url",
```

Comment:
If we end up with many scopes in future I think it would be good to note what the various scopes are for.